nilearn/plotting/img_plotting.py

```diff
@@ -1041,7 +1041,8 @@ def plot_connectome(adjacency_matrix, node_coords,
                     figure=None, axes=None, title=None,
                     annotate=True, black_bg=False,
                     alpha=0.7,
-                    edge_kwargs=None, node_kwargs=None):
+                    edge_kwargs=None, node_kwargs=None,
+                    colorbar=False):
```

Comment:
I prefer to be False since glass brain is overlapped with colorbar as you can see in recent picture I posted. May be some code lines to tweak ?